docs/api-reference/data-fetching/get-static-props.md

```diff
@@ -41,7 +41,7 @@ The `context` parameter is an object containing the following keys:
 
 ## getStaticProps return values
 
-The `getStaticProps` function should return an object with the following **optional** properties:
+The `getStaticProps` function should return object with **any one of the following** properties:
```

Comment:
```suggestion
The `getStaticProps` function should return an object with **any one of the following** properties:
```

Reply:
Will add this in the next changes for both getStaticProps and getServerSideProps